### PR TITLE
Adding domain and enpoints

### PR DESCRIPTION
**Pull request by pploszczyca**
- Dodałem przykładową domenę i endpointy,
- Dodałem swaggera, który znajduje się pod adresem: http://localhost:8080/swagger-ui/index.html#/

### CODE REVIEW

File: backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.pk.zbtz.zbtzbackend.domain
+
+data class Movie(
+    val id: Long,
+    val title: String,
+    val directorId: Long,
```
Comment (Pirgo) [inline backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt]: It should be a list, there can be more than one director in the film.

File: backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    val genreId: Long,
+    val productionYear: Int?,
+    val rating: Float?
```
Comment (Pirgo) [inline backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt]: I think it should also include some description about the movie. Check my project for inspiration

File: backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Director.kt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.pk.zbtz.zbtzbackend.domain
+
+data class Director(
+    val id: Long,
+    val firstName: String,
+    val lastName: String
```
Comment (Pirgo) [inline backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Director.kt]: I would add image here (we should be able to show his face somehow). Birthday would also be nice to have

File: backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt (hunk 2 above)
Comment (Pirgo) [reply backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt]: Also I think we should include info about actors somehow. Probably separete table in DB

File: backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package com.pk.zbtz.zbtzbackend.domain
+
+data class Movie(
+    val id: Long,
+    val title: String,
+    val directorId: Long,
+    val genreId: Long,
```
Comment (Pirgo) [inline backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt]: this should also be a list

File: backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt (hunk 1 above)
Comment (pploszczyca) [reply backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt]: Changed

File: backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt (hunk 2 above)
Comment (pploszczyca) [reply backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt]: Added `plot` and `actors`

File: backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Director.kt (hunk 3 above)
Comment (pploszczyca) [reply backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Director.kt]: Added url with photo

File: backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt (hunk 4 above)
Comment (pploszczyca) [reply backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt]: Changed

**Pull request merged**